.dprint.json

```diff
@@ -8,6 +8,9 @@
   "json": {
     "deno": true
   },
+  "yaml": {
+    "quotes": "preferSingle"
```

Comment:
Used this because this is how deno_std prints yaml in some of our code generated yaml.